docs/usage/self-hosted-configuration.md

```diff
@@ -696,6 +696,10 @@ Set this to an S3 URI to enable S3 backed repository cache.
     If you're storing the repository cache on Amazon S3 then you may set a folder hierarchy as part of `repositoryCacheType`.
     For example, `repositoryCacheType: 's3://bucket-name/dir1/.../dirN/'`.
 
+<!-- prettier-ignore -->
+!!! note
+    S3 repository is used as a repository cache (e.g. extracted dependencies) and not a lookup cache (e.g. available versions of dependencies). To keep the later remotely, define [Redis URL](#redisurl).
```

Comment:
I'm not sure if `#redisurl` can be used, or the absolute URL is required instead.